homework/index.js

```diff
@@ -30,17 +34,110 @@
     return elem;
   }
 
+  function renderError(error) {
+    const body = document.body;
+    body.innerHTML = '';
+    createAndAppend('h1', body, { text: error.message, class: `alert-error` });
+  }
+
+  function addRow(tbody, label, value) {
+    const tr = createAndAppend('tr', tbody);
+    createAndAppend('td', tr, { text: `${label}:`, class: 'label' });
+    createAndAppend('td', tr, { text: value });
+  }
+
+  function renderRepos(repoName, ul) {
+    fetchJSON(`${HYF_REPOS_URL}`, (err, repositories) => {
+      if (err) {
+        renderError(err);
+        return;
+      }
+      ul.innerHTML = '';
+      repositories.forEach(repo => {
+        if (repo.name === repoName) {
+          const li = createAndAppend('li', ul);
+          const table = createAndAppend('table', li);
+          const tbody = createAndAppend('tbody', table);
+          const tr1 = createAndAppend('tr', tbody);
+          createAndAppend('td', tr1, { text: `Name:`, class: 'label' });
+          const a = createAndAppend('a', tr1, { href: `${repo.html_url}` });
+          createAndAppend('td', a, { text: `${repo.name}` });
+          addRow(tbody, 'Description', `${repo.description}`);
+          addRow(tbody, 'Forks', `${repo.forks}`);
+          addRow(tbody, 'Updated', `${repo.updated_at.slice(0, 10)}`);
+        }
+      });
+    });
+  }
+
+  function renderContributions(repoName, ul) {
+    fetchJSON(
+      `https://api.github.com/repos/HackYourFuture/${repoName}/contributors`,
+      (err, repositories) => {
+        if (err) {
+          renderError(err);
+          return;
+        }
+        ul.innerHTML = '';
+        createAndAppend('h3', ul, { text: `Contributors:` });
+        repositories.forEach(repo => {
+          const a = createAndAppend('a', ul, { href: `${repo.html_url}` });
```

Comment:
The `a` tag link should open in a new browser tab, as per the assignment,

Reply:
You are right thanks i added it